.github/workflows/update-sdk-versions.yml

```diff
@@ -28,87 +32,186 @@ jobs:
+          REPO_LIST=$(gh api orgs/growthbook/repos --paginate --jq '.[].full_name')
+          
+          # Check if source repo exists in the organization
+          if echo "$REPO_LIST" | grep -q "^$SOURCE_REPO$"; then
+            echo "✅ Source repository '$SOURCE_REPO' validated"
+          else
+            echo "❌ Error: Repository '$SOURCE_REPO' not found in growthbook organization"
+            echo "Available repositories:"
+            echo "$REPO_LIST"
             exit 1
           fi
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Create Branch
+        run: |
+          SDK="${{ steps.params.outputs.sdk }}"
+          VERSION="${{ steps.params.outputs.version }}"
+          BRANCH_NAME="update-${SDK}-sdk-${VERSION}"
+          
+          git config user.name "github-actions"
+          git config user.email "github-actions@github.com"
+          git checkout -b "$BRANCH_NAME"
+          
+          echo "branch_name=$BRANCH_NAME" >> $GITHUB_ENV
+
+      - name: Update SDK Version File
+        run: |
+          SDK="${{ steps.params.outputs.sdk }}"
+          VERSION="${{ steps.params.outputs.version }}"
+          SDK_FILE="packages/shared/src/sdk-versioning/sdk-versions/${SDK}.json"
           
-          # Navigate to the SDK versions directory
-          cd packages/shared/src/sdk-versioning/sdk-versions
+          echo "Updating SDK version in file: $SDK_FILE"
           
-          # Check if the SDK file exists
-          SDK_FILE="${SDK}.json"
+          # Check if SDK file exists
           if [ ! -f "$SDK_FILE" ]; then
-            echo "Error: SDK file $SDK_FILE does not exist"
+            echo "❌ Error: SDK file $SDK_FILE does not exist"
             exit 1
           fi
           
           # Check if version already exists
           if jq -e --arg v "$VERSION" '.versions[] | select(.version == $v)' "$SDK_FILE" > /dev/null; then
-            echo "Version $VERSION already exists in $SDK_FILE"
-            echo "No update needed - exiting successfully"
-            exit 0
+            echo "⚠️  Version $VERSION already exists in $SDK_FILE"
+            echo "version_file_updated=false" >> $GITHUB_ENV
+          else
+            echo "Adding version $VERSION to $SDK_FILE"
+            jq --arg v "$VERSION" '.versions = ([{"version": $v}] + .versions)' "$SDK_FILE" > "${SDK_FILE}.tmp"
+            if [ $? -eq 0 ]; then
+              mv "${SDK_FILE}.tmp" "$SDK_FILE"
+              git add "$SDK_FILE"
+              echo "version_file_updated=true" >> $GITHUB_ENV
+            else
+              echo "❌ Error: Failed to update $SDK_FILE"
+              rm -f "${SDK_FILE}.tmp"
+              exit 1
+            fi
           fi
+
+      - name: Update SDK Documentation
+        run: |
+          SDK="${{ steps.params.outputs.sdk }}"
+          VERSION="${{ steps.params.outputs.version }}"
+          SDKINFO_FILE="docs/src/data/SDKInfo.json"
```

Comment:
Ryan just changed this recently to be a `.ts` file instead of `.json`. There's a script to re-generate it, but might be easiest to just remove this step for now to unblock you.